### PR TITLE
Preparation for publication

### PR DESCRIPTION
**Pull request by egekorkan**
I have done some work to prepare the publication to npm. I have applied eslint and prettier rules. There is one node-red callback that raises eslint errors and I have added ignore comments there. @hidetak if you can have a look at it, it would be nice (I am adding a pull request comment to it).  Also, please review this PR.

I am also tagging others who were in the discussion last week: @relu91 , @danielpeintner , @JKRhb .
Note: When reviewing, please look at the last commit separately (prettying).

### CODE REVIEW

File: node-red-node-wot/src/wot-property.js [hunk 1]
```diff
@@ -1,174 +1,176 @@
...
+            // eslint-disable-next-line
+            if (removed) {
+                // This node has been deleted
+            } else {
+                // This node is being restarted
+            }
+            // eslint-disable-next-line
+            done()
+        })
+    }
```
Comment (egekorkan) [inline node-red-node-wot/src/wot-property.js]: @hidetak please let me know how I can remove the eslint ignore comments.

File: node-red-node-wot/src/wot-property.js [hunk 2]
```diff
@@ -1,174 +1,176 @@
...
+                        text: "Response error",
+                    })
+                    done(err)
+                })
+        })
 
-  function writePropertyNode(config) {
-    RED.nodes.createNode(this, config)
-    let node = this
+        node.on("close", function () {
```
Comment (hidetak) [inline node-red-node-wot/src/wot-property.js]: Sorry, I must have missed it.
As with other nodes, the arguments must be `removed` and `done`.
```
        node.on("close", function (removed, done) {
```

File: node-red-node-wot/src/wot-property.js (hunk 1 above)
Comment (hidetak) [reply node-red-node-wot/src/wot-property.js]: @egekorkan 
The callback arguments `removed` and `done` were missing.
My apologies for missing that.

https://github.com/eclipse-thingweb/node-red/pull/4/files/ab1be6a993ae86e8f666114d5f687c7e40aa2092#diff-52388816cc91b77d02014cdacab7478e4777a58db9195c95c1b9ad3baf295e15R116

**Comment by hidetak** [comment]
@egekorkan 
I noticed that it is better to terminate the server upon the close event of a node, so I fixed it.
The following PR has been created for this branch
https://github.com/eclipse-thingweb/node-red/pull/5
I apologize for the trouble, but I would appreciate it if you could check it.

File: node-red-node-wot/src/wot-property.js (hunk 1 above)
Comment (hidetak) [reply node-red-node-wot/src/wot-property.js]: @egekorkan 

The following PRs have been corrected together.
https://github.com/eclipse-thingweb/node-red/pull/5/files#diff-52388816cc91b77d02014cdacab7478e4777a58db9195c95c1b9ad3baf295e15R116

File: node-red-node-wot/src/wot-property.js (hunk 2 above)
Comment (egekorkan) [reply node-red-node-wot/src/wot-property.js]: Done via #5 . Thanks!

File: node-red-node-wot/src/wot-property.js (hunk 1 above)
Comment (egekorkan) [reply node-red-node-wot/src/wot-property.js]: Thanks!

**Comment by egekorkan** [comment]
No more eslint warnings and prettier changes. @hidetak if you approve it, I will merge this PR.

**Review by hidetak — APPROVED** [review]

**Pull request merged**